### DISCUSSION TITLE
My account has been blocked for no reason.

### DISCUSSION THREAD

**Discussion by Marchuk98** [vercel/community#4442] [Account Issue]
### Question

Two weeks ago, I reached out to [registration@vercel.com](mailto:registration@vercel.com) regarding an issue with authorizing my git account or registering using my git email. I've received no response. Initially, I thought the problem might be with my email domain, but it seems that's not the case. When I try to register any account, I never receive the confirmation email, despite being told "Thank you, we're glad to have you". And judging by the discussions I've seen, I doubt I'll get much assistance, as it appears that registration issues are either being ignored or are met with generic links to documentation. Kudos to the service.
![git2](https://github.com/vercel/community/assets/81834571/60ad2563-d2b5-474c-89ba-74c072ed4f04)
![git1](https://github.com/vercel/community/assets/81834571/96b5f995-fa7b-4422-91b5-adcc93017967)

**Comment by srcnalt** [comment]
~Could be due to using an IP address located in Russia, I do not know what is the case with Vercel but remember some colleagues having issues with other USA based services blocking accounts and cutting services. Creating a new account using VPN might be a solution.~

I was just checking twitter for clues and saw some people claim to be blocked due to use of VPN. That might also be the case.

**Comment by amyegan** [comment]
You did the right thing by sending that email to open a support case. I know this is a frustrating situation. The team is working to implement a better solution for these login issues. 

Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the cases that were opened before yours. When handling a high number of tickets, as the team is doing now, wait times can get a bit long. 

You can find more information about response times at https://vercel.com/guides/vercel-support-queue-time

**Reply by amyegan** [reply]
It looks like the support team responded to your ticket and unblocked your account. Please let us know if you still have any trouble with your account.
